libraries/abstractions/common_io/test/test_iot_spi.c

```diff
@@ -102,12 +106,12 @@ TEST_GROUP_RUNNER( TEST_IOT_SPI )
     RUN_TEST_CASE( TEST_IOT_SPI, AFQP_IotSPI_CancelFail );
     RUN_TEST_CASE( TEST_IOT_SPI, AFQP_IotSPI_CancelSuccess );
 
-    //RUN_TEST_CASE( TEST_IOT_SPI, AFQP_IotSPI_ReadSync );
-    //RUN_TEST_CASE( TEST_IOT_SPI, AFQP_IotSPI_ReadAsync );
-    //RUN_TEST_CASE( TEST_IOT_SPI, AFQP_IotSPI_WriteSync );
-    //RUN_TEST_CASE( TEST_IOT_SPI, AFQP_IotSPI_WriteAsync );
-    //RUN_TEST_CASE( TEST_IOT_SPI, AFQP_IotSPI_TransferSync );
-    //RUN_TEST_CASE( TEST_IOT_SPI, AFQP_IotSPI_TransferAsync );
+    RUN_TEST_CASE( TEST_IOT_SPI, AFQP_IotSPI_ReadSync );
+    RUN_TEST_CASE( TEST_IOT_SPI, AFQP_IotSPI_ReadAsync );
```

Comment:
These tests require a SPI slave, and are run with assisted test framework. By enabling these here, how do we expect them to run?

Reply:
I will add the #define config so they can be toggled on or off.